flink-contrib/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBMapState.java

```diff
@@ -382,11 +342,26 @@ private UV deserializeUserValue(byte[] rawValueBytes) {
 			this.rawValueBytes = rawValueBytes;
 			this.deleted = false;
 		}
-		
+
+		public void remove() {
+			deleted = true;
+			rawValueBytes = null;
+
+			try {
+				db.remove(columnFamily, writeOptions, rawKeyBytes);
+			} catch (RocksDBException e) {
+				throw new RuntimeException("Error while removing data from RocksDB.", e);
```

Comment:
The intention of my comment about `RuntimeException` was not about changing the method signatures for throwing RocksDBExcepion. My suggestion was to only use a proper subclass of `RuntimeException`. We should avoid using `RuntimeException` directly, similar to how we should avoid throwing the class `Exception` directly. I know that there is some code in Flink that does not follow this, but I think it is better code style to stick with more appropriate subclasses.